users/models.py

```diff
@@ -131,3 +131,17 @@ def save(self, **kwargs):
             self.last_vote_affirmation = timezone.now()
 
         return super().save(**kwargs)
+
+
+class UserGroup(models.Model):
+    name = models.CharField(max_length=255)
+    location = models.CharField(blank=True, null=True, max_length=255)
+    url = models.URLField('URL', blank=True, null=True)
+    url_type = models.IntegerField(choices=(
+        (0, 'Meetup'), 
+        (1, 'Distribution List'), 
+        (2, 'Other')), 
+        null=True)
```

Comment:
I think this field should be required too.

Reply:
Done.